modules/ROOT/pages/api-groups-promote-api-instance.adoc

```diff
@@ -0,0 +1,26 @@
+= Promote API Instances in API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can promote an API Group instance from one environment to another, for example, Sandbox to Production.
+
+Promoting an API Group instance affects only the API definition, not the existing deployed applications subscribed to the API Group. After promoting an API Group instance, the API Name remains the same, but the API Version changes.
+
+Because each environment has a different client ID and secret, you must update the application’s client ID and secret.
+
+// You cannot prompte a API Group instnace if every API instnace is that group is not present in the Production environment also.
```

Comment:
This sentence nearly correct, you must add it. 
```suggestion
// You cannot promote an API Group instance if every API in that group does not have an API instance in any environment of the same environment type that the destination enviroment.
```

Reply:
Done! Thanks!